docs/code-quality/clang-tidy.md

```diff
@@ -31,14 +31,51 @@ For more information, see [How to: Set Code Analysis Properties for C/C++ Projec
 
 ## CMake
 
-In CMake projects, you can configure Clang-Tidy checks within *`CMakeSettings.json`*. Once opened, select "Edit JSON" in the top right-hand corner of the CMake Project Settings Editor. The following keys are recognized:
+In CMake projects, you can configure Clang-Tidy checks within *`CMakeSettings.json`* or *`CMakePresets.json`*. 
+
+The following keys are recognized for clang-tidy:
 
 - `enableMicrosoftCodeAnalysis`: Enables Microsoft Code Analysis
 - `enableClangTidyCodeAnalysis`: Enables Clang-Tidy analysis
 - `clangTidyChecks`: Clang-Tidy configuration, specified as a comma-separated list, that is, checks to be enabled or disabled
```

Comment:
Would you mind fixing the old clangTidyChecks description while you are in this file? Something like: "Clang-Tidy configuration. A comma-separated list of checks to be enabled or disabled."  And could we provide a simple example? How do you specificy enabled or disabled? + or -?